Test has a 98% probability = positive result and sufferer, and 10% probability (false) positive and non-sufferer.
0.5 % of the population are sufferers. 

1. Be positive

In [11]:
0.98*0.005 + 0.1*.955

0.1004

2. Correctly diagnose a sufferer of Thripshaw's

P(A | B) = P(B | A)  *  P(A) / [P(A)* P(B|A) + P(A~) * P(B|A~)]

In [12]:
0.98

0.98

3. Correctly identify a non-sufferer of Thripshaw's


In [8]:
(0.90*0.995)/(0.90*0.995+0.005*0.02)

0.9998883430102724

4. Misclassify the person

negative when sick 

In [14]:
(0.02*0.005)/(0.02*0.005 + 0.005*0.98)

0.02

positive and not sick = 0.10

Misclassify the person through negative: 1 - being correct in both cases
    
    1-(.98*.005 + .9*.955)=.1356